optuna/study.py

```diff
@@ -149,6 +149,15 @@ def trials(self) -> List[FrozenTrial]:
 
         return self.get_trials(deepcopy=True, states=None)
 
+    def _is_multi_objective(self) -> bool:
+        """Return True if the study has multi objectives.
```

Comment:
```suggestion
        """Return :obj:`True` if the study has multi objectives.
```

We usually annotate an object with `:obj:`.

ref. https://github.com/optuna/optuna/blob/e2e2a5cd91f3df967838cea525b28c49993b6721/optuna/storages/_base.py#L338